tools/monorepo-utils/src/index.ts

```diff
@@ -9,12 +9,39 @@ import chalk from 'chalk';
  * Internal dependencies
  */
 import CodeFreeze from './code-freeze/commands';
+import Slack from './slack/commands/slack';
+import { Logger } from './core/logger';
+import { isGithubCI } from './core/environment';
 
-console.log(
-	chalk.rgb( 150, 88, 138 ).bold( figlet.textSync( 'WooCommerce Utilities' ) )
-);
+if ( ! isGithubCI() ) {
+	Logger.notice(
+		chalk
+			.rgb( 150, 88, 138 )
+			.bold( figlet.textSync( 'WooCommerce \n Utils' ) )
+	);
+}
 
-export const program = new Command()
+const program = new Command()
 	.name( 'utils' )
 	.description( 'Monorepo utilities' )
-	.addCommand( CodeFreeze );
+	.addCommand( CodeFreeze )
+	.addCommand( Slack );
+
+program.exitOverride();
```

Comment:
This call allows you to customize how you handle exit codes in commander, I use it here so that we don't error when we show a help screen, e.g. via `pnpm utils`.

Reply:
I'm not sure if its this change or somewhere else, but an error's stack trace is no longer shown. Is this desired behaviour?

Reply:
> I'm not sure if its this change or somewhere else, but an error's stack trace is no longer shown. Is this desired behaviour?

Yeah this will be making that change because Logger just logs the `error.message`.

Stack traces could be useful yes. I think we could probably integrate that into the `Logger` functionality here. I'll have a think about what's nice to do with that.

Reply:
@psealock added stack to error output here https://github.com/woocommerce/woocommerce/pull/38185/commits/e2324b36d966f174e43f56a92d1411d3e410efe6